packages/astro/src/@types/astro.ts

```diff
@@ -127,21 +128,30 @@ export interface AstroUserConfig {
 
 	/**
 	 * @docs
-	 * @name renderers
-	 * @type {string[]}
-	 * @default `['@astrojs/renderer-svelte','@astrojs/renderer-vue','@astrojs/renderer-react','@astrojs/renderer-preact']`
+	 * @name integrations
+	 * @type {AstroIntegration[]}
+	 * @default `["react", "preact", "svelte", "vue"]`
```

Comment:
Hm, is this the case? I thought each of these need to be applied manually, each with functions rather than strings

Reply:
That's out of date! Great catch, updating now.